File: Contacts.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Contacts
{
    class contact {
        public String phone { get; set; }
        public String name { get; set; }
        public String email { get; set; }

        public contact(String name, String email, String phone){
            this.name = name;
            this.phone = phone;
            this.email = email;
        }
    }

    class Program
    {
        List<contact> ContactList = new List<contact>();

        public Program() {
            addContact("Sheryl", "sheryl.beryl@gmail.com", "0216358285");
            GetInfoByName("Sheryl");
        }

        public void addContact(String name, String email, String phone)
        {
            ContactList.Add(new contact(name, email, phone));
        }

        public void GetInfoByName(String name) {
            foreach(contact c in ContactList) {
                if (c.name==name) {
                    Console.WriteLine(c.email);
                    Console.WriteLine(c.phone);
                    Console.ReadKey();
                }
            }
        }

        static void Main(string[] args)
        {
            new Program();


        }
    }
}

File: Contacts.py
contact_data = []

sheryl = []+["Sheryl Beryl"]+["027543771"]
contact_data.append(sheryl)

def addContact():
    while True:
        print("Please enter their information.")
        nam = input("please input their name:")
        phone = input("please input their phone number:")
        contact = []+[nam]+[phone]
        contact_data.append(contact)
        ans = input("would you like to add another contact? ('y' or 'n')")
        if ans == "n":
            ans1 = input("Would you like to do something else? ('y' or 'n')")
            if ans1 == "y":
                start()
                break
            if ans1 == "n":
                break
        if ans == "y":
            continue 

def search():
    nam = input("Enter the name you want to search for")

    for x in contact_data:
        for contact in x:
            if contact == nam:
                print(contact)
                print(x)

def getAll():
    for x in contact_data:
        for y in x:
            print(y)

    while True:
        choice = input("Would you like to do something else? ('y' or 'n'")
        if choice == "y":
            start()
            break
        if choice == "n":
            break
        else: 
            print ("Please enter a valid command")
            continue 

def start():
    print ("If you want to make a new contact type 'New'")
    print ("If you want to see all contacts type 'All'")
    print ("If you want to search for a contact type 'Search'")

    while True:
        choice = input()
        if choice == "New" or "new":
            addContact()
            break
        if choice == "All" or "all":
            getAll()
            break
        if choice == "Search" or "search":
            search()
            break
        else: 
            print ("Please enter a valid command")
            continue 

print ("Welcome to Contact List 1.0")
start()